communications/cloud.md

```diff
@@ -1,73 +1,91 @@
 
-Where possible, clients should be the account owner of cloud services in client engagements. While the security committee can provide feedback, the client should be primarily responsible for how they want to handle security reviews, if any.
+2. **Data Handling**:
+   - Can data be deleted if needed?
+   - Is data being shared with others?
+   - Are they using your data to train AI models?
+   - How do they handle accidental input of sensitive information?
 
-## When do I use my Lullabot email and when do I use my personal email when signing up for Cloud services?
-- If you are signing up for a service paid for by Lullabot, or a Lullabot client, use the Lullabot email address.
-- If you are signing up for a service at the request of Lullabot, or a Lullabot client, use the Lullabot email address.
-- If you are signing up for a service that will contain or provide access to Lullabot data, or Lullabot client data, use the Lullabot email address.
-- If you are signing up for a service that you would continue to use even if you were no longer employed at Lullabot, use your personal email address.
-- If you are signing up for a service that will not contain or provide access to Lullabot data, nor be used by others at Lullabot or Lullabot clients, use your personal email address.
+3. **Permissions**:
+   - What integrations does the service require (e.g., calendars, email, file storage)?
+   - If it requires more than basic login information, involve the security team for review.
 
-If a service supports multiple email addresses per account, it’s OK (and even recommended) to use one “account” with multiple “identities”. GitHub is the most notable example of this, though services that are built on top of GitHub likely use the same model. For example, instead of creating a new account, add your Lullabot email address to your existing account. Use the built-in tools to associate work and personal email addresses with the right organizations and projects.
+4. **Security History**:
+   - Has the company had any notable security breaches?
+   - Check sources like Hacker News and Google News for reported incidents.
 
-Some clients will create email addresses for you. Where possible, use your Lullabot email address instead of a client email address. This is especially important for VPN recovery, where client email may not be accessible outside of the VPN. Consider redirecting your client email to your Lullabot email if it’s supported by the client.
+### Decision Tree for Evaluation
 
-There are times when the answers might conflict, and you could make a case for using either email address. When in doubt communicate with your manager or the security team for instructions.
+To help guide your evaluation process, we've created a decision tree. You can find the visual representation [here](https://www.mermaidchart.com/raw/9bce760b-17c1-4154-b09f-09b9b955f7ff?theme=light\&version=v0.1\&format=png). 
```

Comment:
Since this links to a PNG, how about we embed it directly?

Reply:
Do you mean upload the file and embed or actually just embed in the .md file?